src/helpers/verifyUnique.ts

```diff
@@ -23,23 +25,39 @@ export const ensureCollection = async ({
         collection = new Collection(id);
         collection.collectionId = collectionIdString;
         collection.isDestroyed = false;
+        collection.issuer = '';
+        collection.owner = '';
+        collection.admin = '';
```

Comment:
Can you please add these fields to `class Collection` and initialise them in the constructor there instead of here?